CONTRIBUTING.md

```diff
@@ -0,0 +1,340 @@
+use the ``reno report`` command. If you want to generate the full Machine Learning release
+notes for all releases (since we started using reno during 0.9) you just run::
+
+    reno report
+
+but you can also use the ``--version`` argument to view a single release (after
+it has been tagged::
+
+    reno report --version 0.5.0
+
+At release time ``reno report`` is used to generate the release notes for the
+release and the output will be submitted as a pull request to the documentation
+repository's [release notes file](
+https://github.com/Qiskit/qiskit/blob/main/docs/release_notes.rst)
+
+#### Building release notes locally
+
+Building The release notes are part of the standard qiskit-machine-learning documentation
+builds. To check what the rendered html output of the release notes will look
+like for the current state of the repo you can run: `tox -edocs` which will
+build all the documentation into `docs/_build/html` and the release notes in
+particular will be located at `docs/_build/html/release_notes.html`
+
+## Installing Qiskit Machine Learning from source
+
+Please see the [Installing Qiskit Machine Learning from
+Source](https://github.com/qiskit-community/qiskit-machine-learning#installation)
+section of the Qiskit documentation.
+
+Note: Machine Learning depends on Terra, and has optional dependence on Aer and IBM Q Provider, so
+these should be installed too. The main branch of Machine Learning is kept working with those other element
+main branches so these should be installed from source too following the instructions at 
+the same location
+
+Machine Learning also has some other optional dependents see 
+[Machine Learning optional installs](https://github.com/qiskit-community/qiskit-machine-learning#optional-installs) for
+further information. Unit tests that require any of the optional dependents will check
+and skip the test if not installed.
+
+### Testing
+
+Once you've made a code change, it is important to verify that your change
+does not break any existing tests and that any new tests that you've added
+also run successfully. Before you open a new pull request for your change,
+you'll want to run the test suite locally.
+
+The test suite can be run from a command line or via your IDE. You can run `make test` which will
+run all unit tests. Another way to run the test suite is to use
+[**tox**](https://tox.readthedocs.io/en/latest/#). For more information about using tox please
+refer to
+[Terra CONTRIBUTING](https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#test)
+Test section. However please note Machine Learning does not have any
+[online tests](https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#online-tests)
+nor does it have
+[test skip
+ options](https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#test-skip-options).    
+
+### Development Cycle
+
+The development cycle for qiskit-machine-learning is informed by release plans in the 
```

Comment:
```suggestion
The development cycle for qiskit-algorithms is informed by release plans in the 
```

Reply:
I have no idea going forwards with this - ie why a community project would have rfcs in Qiskit organization. Same is true for apps.

Reply:
Well, there is at least one, the one about the repo migration

Reply:
Right but then it was talking about code in the Qiskit org... now, no more. Its not of course all that important in the scheme of things at present - just figured I comment while I saw it.